aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -372,4 +408,33 @@ protected ProgressEvent<ResourceModel, CallbackContext> removeFromGlobalCluster(
                 ))
                 .progress();
     }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> setDefaultVpcSecurityGroupIds(
+            final AmazonWebServicesClientProxy proxy,
+            final ProxyClient<RdsClient> rdsProxyClient,
+            final ProxyClient<Ec2Client> ec2ProxyClient,
+            final ProgressEvent<ResourceModel, CallbackContext> progress
+    ) {
+        final ResourceModel resourceModel = progress.getResourceModel();
+        SecurityGroup securityGroup;
+        try {
+            DBCluster cluster = fetchDBCluster(rdsProxyClient, resourceModel);
+            DBSubnetGroup subnetGroup = fetchDBSubnetGroup(rdsProxyClient, cluster.dbSubnetGroup());
+            securityGroup = fetchSecurityGroup(ec2ProxyClient, subnetGroup.vpcId(), "default");
+            if (securityGroup != null) {
+                resourceModel.setVpcSecurityGroupIds(Lists.newArrayList(securityGroup.groupId()));
+            }
+            return progress;
+        } catch (Exception exception) {
+            return Commons.handleException(
+                    ProgressEvent.progress(resourceModel, progress.getCallbackContext()),
+                    exception,
+                    DEFAULT_DB_CLUSTER_ERROR_RULE_SET);
```

Comment:
Do we need to expand error rule set to accommodate for potential EC2 errors?

Reply:
Done.